### DISCUSSION TITLE
ghippo/install/offline-install/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1413] [Announcements]
# ghippo/install/offline-install/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/ghippo/install/offline-install/

**Comment by panpan0000** [comment]
`global.database.apiserver.use` 这些不需要写成变量，因为安装器装出来用户名和库之类的都是钉死的。手动改太麻烦了

**Reply by yitingdc** [reply]
新的版本已经没有这些变量了，已经改成dsn了
